README.md

```diff
@@ -21,14 +21,13 @@ bun install
 
 Finally, create a .env file with the `TOKEN` and `GUILD` variables. `TOKEN` should be set to the token of your Discord bot account and `GUILD` should be set to the id of the guild you want the bot to function in (this bot is configured by default to only run in a single server).
 
-It also need a CSE API key and a CSX if you want `/mdn` to work. It can optionally have a `SCRAPE_CACHE` which is the duration in hours of the scrape cache. Defaults to 1 hour.
+It also need a CSE API key and a CSX if you want `/mdn` to work. It can optionally have a `SCRAPE_CACHE_LIFETIME` which is the duration in hours of the scrape cache. Defaults to 1 hour.
```

Comment:
This isn't really necessary anymore (at least the CSE/CSX env var notes).

Reply:
Ah, correct, I did miss those, even though I _did_ to a project-wide search. Hm.